torch/_dynamo/utils.py

```diff
@@ -758,7 +758,7 @@ def wrap_to_fake_tensor(e, fake_mode):
 
 
 def wrap_to_fake_tensor_and_record(e, tx):
-    if type(e) in (torch.Tensor, torch.nn.Parameter):
+    if isinstance(e, torch.Tensor) and not isinstance(e, torch._subclasses.FakeTensor):
```

Comment:
This seems questionable; shouldn't we be guaranteed to NOT have fake tensors when we're wrapping

Reply:
This is for another reason, TensorProxy